gatsby-config.js

```diff
@@ -32,5 +32,7 @@ module.exports = {
     `gatsby-plugin-sharp`,
     `gatsby-transformer-remark`,
     `gatsby-plugin-sass`,
+    `gatsby-transformer-json`,
```

Comment:
What is the purpose of `gatsby-transformer-json`?

Reply:
It was recommended for adding in a more variable title but I may be able to remove that and still have the React Helmet work

Reply:
Removed the JSON plugin and the React Helmet does still work without it so I will keep that out for now.